src/server/modes/charts/plugins/datalens/preparers/metric/variants/markup.ts

```diff
@@ -0,0 +1,48 @@
+import {
+    MarkupItem,
+    ServerCommonSharedExtraSettings,
+    ServerField,
+} from '../../../../../../../../shared';
+
+export const prepareMarkupMetricVariant = ({
+    measure,
+    value,
+    extraSettings,
+}: {
+    measure: ServerField;
+    value: string | MarkupItem | null;
+    extraSettings: ServerCommonSharedExtraSettings | undefined;
+}) => {
+    if (typeof value === 'string') {
+        const size = (extraSettings && extraSettings.metricFontSize) || 'm';
+        const color = (extraSettings && extraSettings.metricFontColor) || 'rgb(77, 162, 241)';
+
+        return {
+            value: {
+                type: 'concat',
+                className: `markup-indicator_size_${size}`,
+                children: [
+                    {
+                        className: 'markup-indicator-title',
+                        type: 'text',
+                        content: measure.fakeTitle || measure.title,
```

Comment:
The chart title is a separate setting. I don't think it should be a part of the markup.

Reply:
This part code is intended to be a replacement for old metric. Old metric had title as a part of it. But i've added now correct setting pass, to support title from extraSettings.